src/librustdoc/passes/html_tags.rs

```diff
@@ -207,7 +208,7 @@ impl<'a, 'tcx> DocFolder for InvalidHtmlTagsLinter<'a, 'tcx> {
 
             for (tag, range) in tags.iter().filter(|(t, _)| {
                 let t = t.to_lowercase();
-                ALLOWED_UNCLOSED.iter().find(|&&at| at == t).is_none()
+                !ALLOWED_UNCLOSED.iter().contains(&t.as_str())
```

Comment:
Better yet, just use the slice [`contains`](https://doc.rust-lang.org/std/primitive.slice.html#method.contains).

There's another instance with `ALLOWED_UNCLOSED` like this in `fn drop_tag`.